internal/vendor/wiz.go

```diff
@@ -141,10 +143,28 @@ var Environment = []string{
 type ProjectCloudOrganizationLinkInput struct {
 	CloudOrganization string         `json:"cloudOrganization"`
 	Environment       string         `json:"environment"` // enum Environment
+	ResourceGroups    []string       `json:"resourceGroups"`
```

Comment:
In the GraphQL schema, ResourceGroups is optional.

```suggestion
	ResourceGroups    []string       `json:"resourceGroups,omitempty"`
```

If there is a need to deviate from the GraphQL schema, please make a comment in the struct comments.